public/static/locales/en/me.json

```diff
@@ -247,5 +247,8 @@
   "apiKeyMessage2": "Once you generate an API Key, we will require a two factor authentication for your login. This you’ll be able to setup next time you log in.",
   "apiKeyMessage3": "Regenerating an API Key will invalidate your existing API Key immediately.",
   "regenerateKey": "Regenerate Key",
-  "giftFund": "GiftFund"
-}
\ No newline at end of file
+  "giftFund": "GiftFund",
+  "issueReceipts": "Issue Receipts",
+  "taxReceiptsSuccess": "Successfully generated tax receipts",
+  "taxReceiptsAlreadyGenerated": "All tax receipts has already been generated"
```

Comment:
@mariahosfeld Perhaps the text `taxReceiptsAlreadyGenerated` can let the user know where they can download the receipts as well.

Reply:
Yes, that would be good. @Shreyaschorge , first please change the "has" to "have" and then add "Please check the donation details" to indicate where to find the receipts.